delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/FilesSpec.scala

```diff
@@ -55,8 +56,9 @@ class FilesSpec
     with AkkaSourceHelpers
     with RemoteContextResolutionFixture
     with FileFixtures {
-  implicit private val sc: Scheduler = Scheduler.global
-  implicit val ec: ExecutionContext  = system.dispatcher
+  implicit private val sc: Scheduler          = Scheduler.global
+  implicit val ec: ExecutionContext           = system.dispatcher
```

Comment:
We don't need the executionContext in the tests anymore

Reply:
Execution context is needed because `ConfigFixtures` needs it.

Reply:
I've changed it in `ConfigFixtures`